src/Spice86.Core/Emulator/Devices/Cmos/RealTimeClock.cs

```diff
@@ -437,23 +417,14 @@ private void ValidateDivider(byte written) {
     }
 
     private void OnPausing() {
-        if (_isPaused) {
-            return;
-        }
-        _pauseStartedTime = _wallClock.UtcNow;
-        _isPaused = true;
+        _clock.OnPause();
```

Comment:
No, clocks should be registered to pause / resume events independently of this piece of hardware emulation